<launch>

    <arg name="image_src" default="/image_raw" />
    <arg name="camera_info_src" default="/camera_info" />

    <node pkg="image_processor" type="image_rectifier" name="image_rectifier4" output="screen" ns="/camera4">
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>
    <node pkg="image_processor" type="image_rectifier" name="image_rectifier3" output="screen" ns="/camera3">
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>
    <node pkg="image_processor" type="image_rectifier" name="image_rectifier2" output="screen" ns="/camera2">
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>
    <node pkg="image_processor" type="image_rectifier" name="image_rectifier6" output="screen" ns="/camera6">
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>
    <node pkg="image_processor" type="image_rectifier" name="image_rectifier5" output="screen" ns="/camera5">
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>

</launch>